delivery_state/data/mail_template.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
```

Comment:
Of course, as suggested here https://github.com/OCA/delivery-carrier/pull/516#issuecomment-1210379447 another template has been created and the mechanism to send an automatic email when picking has been delivered to the customer (in a separate commit).

Reply:
Yes, it's only I can't see the difference with the core one :sweat_smile:

Reply:
This would be the explanation / difference:
- **Core**: Send an email to the customer when picking is Done (leaves the warehouse).
- **Custom**: Send an email to the customer when the carrier has delivered the package (`delivery_state=customer_delivered`).

Reply:
Updated email template.